src/components/Q&A/QuestionComp.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import AddAnswer from './AddAnswer.jsx';
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+class QuestionComp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questions: this.props.questionList,
+      helpClick: false,
+      showAns: false
+    }
+    this.handleQHelp = this.handleQHelp.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+
+  handleQHelp(qObj) {
+    var temp = parseInt(qObj.question_id);
+    console.log(temp);
+    axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/qa/questions/${temp}/helpful`,
+    {},
+    {headers: {'Authorization': `${API_KEY}`}})
+    .then((res) => {console.log(res)})
+    .catch((err) => {console.log(err)});
+    qObj.question_helpfulness += 1;
+    this.setState({questions: qObj});
+    this.setState({helpClick: true});
+
+}
+
+
+  handleClose(e) {
+    e.preventDefault();
+    this.setState({showAns: false});
+  }
+
+
+//Helpful? {this.state.questions.question_helpfulness}
+  render() {
+    console.log(this.props.proID);
+    let helpBtn
+    if(this.state.helpClick) {
+      helpBtn = <span> Yes! </span>
```

Comment:
Woah, didn't know you could assign JSX to a variable! That's so cool! Definitely will be doing that!

Reply:
It's super helpful when you want to conditionally render something. I use it in almost any button that changes after click.